### ISSUE TITLE
matcaffe error 

### ISSUE THREAD

**Issue by haoliyoupai09** [unsky/Deformable-ConvNets-caffe#24]
After I train the network you provide and obtain the Resnet_dcn.caffemodel, I compiled the matlab interface  and try to load the caffemodel. However, an error appeared (caffe.layerparameter has no field named deformable-cnvolution parameter). I do not know whether it is caused by the incorrect compilation of matcaffe or the loss of the newly added deformable convolutional layer in caffe_.mexa64. Could you help me?

**Comment by unsky** [comment]
sorry, the codes dont support matlab

**Comment by nameToken** [comment]
@haoliyoupai09  you solve the matcaffe error problem?I also want to use deformable conv with matcaffe,looking forward to your reply!!

**Comment by haoliyoupai09** [comment]
Yes, I recompiled the matcaffe interface and it worked. Adding a new layaer in caffe deos not influence the use of caffemodel in matlab.

**Comment by DanChen001** [comment]
@haoliyoupai09   can you use this code on Caffe + Matcaffe successfully?

**Comment by haoliyoupai09** [comment]
Yes, no error occurs during the compilation of matcaffe and running of code. But the performance is not good with deformable convolutional network. I do not know why.

**Comment by DanChen001** [comment]
> Yes, no error occurs during the compilation of matcaffe and running of code. But the performance is not good with deformable convolutional network. I do not know why.

@haoliyoupai09    Thanks.  What  is task you are working on?  Are the performance gap significant or slight?

**Comment by DanChen001** [comment]
> Yes, no error occurs during the compilation of matcaffe and running of code. But the performance is not good with deformable convolutional network. I do not know why.

@haoliyoupai09   I placed one deformable layer at the end of my network. After training, the outputs of this deformable layer are 0. I checked the outputs of the offset layer, which are very large.  Do you know why?  Could you share more details about how to use this layer?  Thanks
